lib/events/badge.js

```diff
@@ -2,11 +2,28 @@
 //
 // Badge events.
 
-import snowplow from '../snowplow';
-import validate from '../validate';
+import stringify  from 'json-stable-stringify';
+import omit       from 'object.omit';
+import snowplow   from '../snowplow';
+import validate   from '../validate';
+
+const CONTENT_TYPES       = ['reviewable', 'customer_experience', 'conversations'];
+const HIT_TYPES           = ['impression', 'non_impression', 'miss'];
+const CTA_PAGE_USES       = ['single_reviewable', 'multi_reviewable'];
+
+// To maintain backward compatibility we try to determine CTA page use when not provided.
+function determineCtaPageUse(opts) {
+  if (opts.ctaPageUse) return opts.ctaPageUse;
+  // It's "multi_reviewable" when badgeVariant value contains term "category" or "listing"
+  if (opts.badgeVariant && opts.badgeVariant.match(/category|listing/)) return 'multi_reviewable';
+  return 'single_reviewable';
```

Comment:
If it's just one line function I would agree. But in this case seems more expressive to put default value on a separate line.